markdown/blog/2025-april-summary.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Monthly Community Update: April 2025"
+date: 2025-04-30T06:00:00+01:00
+type: Communication
+tags:
+  - Project Status
+cover: /img/posts/2025-blog-banner/banner-april.webp
+authors:
+  - name: Thulisile Sibanda
+    photo: /img/avatars/thulieblack.webp
+    link: https://www.linkedin.com/in/v-thulisile-sibanda/
+    byline: Community Builder and Open Source Fanatic!
+excerpt: 'AsyncAPI community status and project updates for April 2025'
+featured: true
+---
+
+We had a busy month, from having our first conference of the year, receiving 204 applications for Google Summer of Code (GSoC'25),  to some sponsorship updates. Let's dive into the April update!
+
+## New Sponsors
+
+We are excited to share that [Mercedes-Benz](https://opensource.mercedes-benz.com/manifesto/) is supporting the initiative as our Bronze sponsor and, most importantly, that they have officially confirmed they use AsyncAPI.
```

Comment:
the prospectus do not offer sharing such info about bronze sponsors.

so only social media posts.

cc @iambami 

<img width="953" alt="Screenshot 2025-04-23 at 09 56 20" src="https://github.com/user-attachments/assets/b2c5e3c3-5ba5-44c1-a02a-cdb817344583" />

Reply:
So I must remove the new sponsors section then?

Reply:
yes, we should follow the rules we setup :)

Reply:
🫡🫡